src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -292,32 +253,25 @@ public static void UnexpectedException_Throws_FaultException()
         Assert.True(false, "Expected FaultException<FaultDetail> exception, but no exception thrown.");
     }
 
-    [Fact]
+    [Theory]
```

Comment:
Got rid of the [Theory]. Sadness that the .NET Native environment doesn't handle it well.